src/Microsoft.DotNet.XHarness.iOS/ErrorKnowledgeBase.cs

```diff
@@ -14,31 +14,39 @@ namespace Microsoft.DotNet.XHarness.iOS
 {
     public class ErrorKnowledgeBase : IErrorKnowledgeBase
     {
-        private static readonly Dictionary<string, string> _testErrorMaps = new Dictionary<string, string>
+        private static readonly Dictionary<string, (string HumanMessage, string IssueLink)> _testErrorMaps = new Dictionary<string, (string HumanMessage, string IssueLink)>
```

Comment:
nit: at this point it might be worth consideration to define alias for this file for `Dictionary<string, (string ....`

Reply:
Believe it or not, I did think about it and I was not sure. The fact that you had the same idea means is a good one. The amount of ? Is that is crazy, but worth it.

Reply:
And we will be getting new `!` with C# 9.0! 😁